PED/AddScenarioBlockingArea.md

```diff
@@ -5,20 +5,24 @@ ns: PED
 
 ```c
 // 0x1B5C85C612E5256E 0xA38C0234
-int ADD_SCENARIO_BLOCKING_AREA(float x1, float y1, float z1, float x2, float y2, float z2, BOOL p6, BOOL p7, BOOL p8, BOOL p9);
+int ADD_SCENARIO_BLOCKING_AREA(float posMinX, float posMinY, float posMinZ, float posMaxX, float posMaxY, float posMaxZ, BOOL network, BOOL cancelActive, BOOL blockPeds, BOOL blockVehicles);
+```
+
+```
```

Comment:
Description should be outside the fenced code blocks since that's reserved for NativeDB imported documentation.